src/Otiport.API/Repositories/Implementations/AdressInformationRepository.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Otiport.API.Data;
+using Otiport.API.Data.Entities.AddressInformations;
+
+namespace Otiport.API.Repositories.Implementations
+{
+    public class AdressInformationRepository : IAdressInformationRepository
+    {
+        private readonly OtiportDbContext _dbContext;
+        private readonly ILogger<IAdressInformationRepository> _logger;
+
+        public AdressInformationRepository(OtiportDbContext dbContext, ILogger<IAdressInformationRepository> logger)
+        {
+            _dbContext = dbContext;
+            _logger = logger;
+        }
+
+        public void Dispose()
+        {
+            _dbContext?.Dispose();
+        }
+
+        public async Task<bool> SaveAsync()
```

Comment:
Eğer bu repository de SaveAsync hiç kullanılmayacaksa not implemented kalabilir. Şuan bir sıkıntı yok sadece bir bilgi